### ISSUE TITLE
release: v20.1.0-beta.2 (expected: 2020-03-02)

### ISSUE THREAD

**Issue by dt** [cockroachdb/cockroach#45173]
Candidate SHA: `49e2b18471a4`
Deployment status:
* http://otan-release-v2010-beta1-2bd6bf44-0002.roachprod.crdb.io:26258/#/overview/list
* http://otan-release-v2010-beta2-49e2b184-0001.roachprod.crdb.io:26258/
Nightly Suite: https://teamcity.cockroachdb.com/viewQueued.html?itemId=1764812
Release Qualification: https://teamcity.cockroachdb.com/viewLog.html?buildId=1763147&buildTypeId=Cockroach_ReleaseQualification
Sentry: https://sentry.io/organizations/cockroach-labs/releases/v20.1.0-beta.2/?project=164528
 
[Release process checklist](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process)
 
Prep date: `2020-02-24`
- [x] [Pick a SHA](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-PickingaSHA)
  - [x] fill in `Candidate SHA` above
  - [x] notify #release-announce
  - [x] email thread on releases@
- [x] [Tag the provisional SHA](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-TagtheprovisionalSHA)
- [x] [Publish provisional binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Publishtheprovisionalbinaries)
 
Release Qualification
- [x] [Check binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Checkbinaries)
- [x] [Deploy to test clusters](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-DeploytoTestClusters)
- [x] [Verify node crash reports](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Verifynodecrashreportsappearinsentry.io)
- [x] [Start release qualification suite](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-RuntheSuiteofNightlyRoachtests)
- [x] [Start nightly suite](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-RuntheSuiteofNightlyRoachtests)
- [x] fill in `Deployment status` above with clusters and `Nightly Suite` with the link to Nightly TeamCity Job
 
One day after prep date:
- [x] [Get signoff on roachtest failures](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-RoachtestFailureSignoff)
 
- [Keep an eye on clusters](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-AppendixA:MonitoringTestClusters) until release date. Do not proceed below until the release date.
 
Release date: `WIP fill in date: usually Monday WIP`
- [x] [Check cluster status](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Checkclusterstatus)
- [x] [Tag release](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Tagtherelease)
- [x] [Bless provisional binaries](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-Blesstheprovisionalbinaries)
 
- For production or stable releases in the latest major release series
  - [x] (omitted as it is unstable) Update the [Homebrew Formula](https://github.com/cockroachdb/homebrew-tap/blob/master/Formula/cockroach.rb)
  - [x] (omitted as it is unstable) Update the image tag in our [Orchestrator configurations](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Orchestratorconfigurations)
  - [x] (omitted as it is unstable) Update the image tag in Helm's [Orchestrator configurations](https://wiki.crdb.io/wiki/spaces/CRDB/pages/73105625/Release+process#Releaseprocess-Orchestratorconfigurations)
  - [x] (omitted as it is unstable) Announce version to the [registration cluster](https://cockroachlabs.atlassian.net/wiki/spaces/ENG/pages/73105625/Release+process#Releaseprocess-AnnounceVersionToRegCluster)
 
- [x] Update docs
- [ ] External communications for release
- [x] Clean up provisional tag from repository

**Comment by otan** [comment]
# Test failures signoff

## VModule Nightly
All tests passed, TC parse misconfig error instead.

## Random Syntax Tests
Allow to fail

## Roachtest GCE
https://teamcity.cockroachdb.com/viewLog.html?buildId=1764809&tab=buildResultsDiv&buildTypeId=Cockroach_Nightlies_WorkloadNightly

## kv
- [] gossip/chaos/nodes=9
- [] scaledata/filesystem_simulator/nodes=6
- [] copy/bank/rows=10000000,nodes=9,txn=false
- [] copy/bank/rows=10000000,nodes=9,txn=true
- [] hotspotsplits/nodes=4

## appdev
- [] django

## bulk-io
- []  schemachange/mixed/tpcc

## auto accept
sqlsmith/setup=empty/setting=default
sqlsmith/setup=rand-tables/setting=default
sqlsmith/setup=seed/setting=default
sqlsmith/setup=seed/setting=no-ddl
sqlsmith/setup=tpcc/setting=default
sqlsmith/setup=tpcc/setting=no-ddl
sqlsmith/setup=tpcc/setting=no-mutations
sqlsmith/setup=tpch-sf1/setting=default

**Comment by tbg** [comment]
Signing off on

gossip/chaos/nodes=9
copy/bank/rows=10000000,nodes=9,txn=false
copy/bank/rows=10000000,nodes=9,txn=true
hotspotsplits/nodes=4

These are known flaky (hotspotsplits is about to get fixed, copy needs a small adjustment after change of default range size, and the chaos test has a repro cycle but we haven't gotten root cause yet).

**Comment by apantel** [comment]
Signing off on django, it's an issue with the number of temp tables used

**Comment by nvanbenschoten** [comment]
Signing off on `scaledata/filesystem_simulator/nodes=6` - we've seen the `Aborting Retries because this error of type *errors.errorString is not retryable : unexpected EOF` error before.

**Comment by otan** [comment]
from @dt on schemachange/mixed/tpcc

> Sure, let’s sign off. Ideally I’d treat this as a blocker until we really dig in and get the root cause, but i’m not sure we will here and don’t we will be able to do that right now and I don’t think we have clear evidence this is a new issue or regression, since we’ve seen similar timeout from time to time before, so I don’t think we can call it a blocker for the beta.

**Comment by otan** [comment]
@rmloveland @Amruta-Ranade this is now ready for docs

**Comment by rmloveland** [comment]
(X-post from the release mailing list)

The release notes are now live on the website:

https://www.cockroachlabs.com/docs/releases/v20.1.0-beta.2.html

**Comment by otan** [comment]
considering issue closed!
